CHANGELOG.md

```diff
@@ -4,9 +4,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
-- Dynamic generation of CSV (Sample,template) file for implemented submodule. 
-- New api for transaction history download function for each process id.
-- Create api to support dynamic submodule help page.
+- Error handling & input validation Messages for exceptions during upload / creation.
+- Manufacturer country code list.
+- Cancel contract agreement on provider side.
+- BPN validation in SDE.
+
+## [1.9.0] - 2023-02-28
+### Added
+- Added new submodel SingleLevelUsageAsBuilt 1.0.1 aspect.
+- Updated Batch submodel to support 1.0.2 version.
+- Updated SerialPartTypization in Version 1.1.0 submodel support.
+- Updated AssemblyPartRelationship in Version 1.1.1 submodel support.
+- Added Look-Up process for AssemblyPartRelationship submodel.
+
+### Fixed
+- Bug fixes.
```

Comment:
Which bugs has been fixed? That's what the Changelog is for.

Reply:
I asked team to add description for bugs, Work in progress.

Reply:
Added description.